sktime/performance_metrics/annotation/__init__.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+
+"""Metrics to assess performance of segmentation task."""
+
+from sktime.performance_metrics.annotation.metrics import (  # noqa
+    annotation_error,
+    hausdorff_error,
+    prediction_ratio,
+)
+
+__author__ = ["lmmentel"]
```

Comment:
do we want an `__all__` variable in here?

(there must be an automatic way of creating the __all__ variable based on the imports, right??)

Reply:
`__all__` would be redundant since it would have all three functions which is equivalent to `*`